docs/developer/architecture/service-observability.md

```
@@ -0,0 +1,116 @@
+
+### What is a Tracer?
+
+A tracer is the component responsible for creating spans. Each service or component can use a tracer (usually named) to start spans consistently. All spans are created using the global tracer provider.
+
+---
+
+## Tracer Initialization
+
+All services should call `InitTracer()` during startup to configure the global OpenTelemetry tracer:
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)